### PR TITLE
Update rabbitmq user password without recreation

### PR DESCRIPTION
**Pull request by puzan**
##### SUMMARY

Current rabbitmq_user module implementation recreates user for password update. It leads to unnecessary actions on target host (delete, create, set tags and permissions). `rabbitmqctl` has method for changing password: `change_password`. This PR provides rabbitmq_user module update with using this method.

New module param is added for password update: `update_password`. `force` is still used for user recreation.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
Module: rabbitmq_user

##### ANSIBLE VERSION
ansible 2.3.0.
Develop: ea0bae6bcaff768c9dc4f2de1b8efbe76b5e89a5

##### ADDITIONAL INFORMATION
Sample playbook:

```yml
- name: Test rabbit
  hosts: test
  gather_facts: False
  tasks:
    - name: rabbit user
      rabbitmq_user:
        user: test
        password: test
        vhost: /
        tags: administrator
        configure_priv: .*
        read_priv: .*
        write_priv: .*
        state: present
        update_password: yes
```

### CODE REVIEW

File: lib/ansible/modules/messaging/rabbitmq_user.py [hunk 1]
```diff
@@ -100,6 +100,13 @@
     required: false
     default: "no"
     choices: [ "yes", "no" ]
+  update_password:
+    description:
+      - Update user password withot user recreation
```
Comment (romanek-adam) [inline lib/ansible/modules/messaging/rabbitmq_user.py]: withot -> without

**Comment by puzan** [comment]
ready_for_review

**Comment by romanek-adam** [comment]
shipit

**Comment by krzysztof-magosa** [comment]
shipit

**Comment by puzan** [comment]
bot_status

**Comment by puzan** [comment]
bot_status

**Comment by puzan** [comment]
ready_for_review

**Comment by puzan** [comment]
Already done in #36280

**Pull request closed without merging**